index.html

```diff
@@ -217,5 +217,19 @@ <h2>
             </div>
         </section>
         </main>
+
+        <footer> 
+            <section class="contact">
+                <h2>Contact me</h2>
+              <p>I'm always interested in hearing about new projects,
+              so if you'd like to chat please get in touch.</p>
+              <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
:red_circle:  Your form should submit successfully the data to formspree. Please make sure your set up an account and copy the right link. For now, your form doesn't exist on formspree

![notsubmitted](https://user-images.githubusercontent.com/26736582/199836246-24738a2a-ebfb-46b6-b0bb-58da61f7e353.gif)

- [ ] This `gif` shows how you can create a form on formspree. Copy your  link and replace this with it 
![copy link](https://user-images.githubusercontent.com/26736582/199836672-e3dbd59e-2a1b-4d91-b137-34bf2161ef37.gif)